portfolio/src/main/java/com/google/sps/servlets/HelloWorldServlet.java

```diff
@@ -10,9 +27,40 @@
 @WebServlet("/hello")
 public class HelloWorldServlet extends HttpServlet {
 
+  private final Date startTime = new Date();
+
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello world!</h1>");
+    // Calculate server stats
+    Date currentTime = new Date();
+
+    // Convert data to JSON
+    MarcoStats marcoStats = new MarcoStats(20, "Marco");
```

Comment:
When passing literal values into methods or constructors, it can help readability to add inline comments to describe the values' meanings: `new MarcoStats(/*age=*/20, /*name=*/"Marco")`. Some automated tooling can actually use these comments to check against the param names and catch potential bugs: https://clang.llvm.org/extra/clang-tidy/checks/bugprone-argument-comment.html#bugprone-argument-comment.